CRM/Contribute/BAO/ContributionRecur.php

```diff
@@ -274,6 +272,11 @@ public static function cancelRecurContribution($params, $activityParams = []) {
     if (!$recurId) {
       return FALSE;
     }
+    $activityParams
```

Comment:
Shouldn't these 2 lines be on one? @eileenmcnaughton

Reply:
I thought jenkins was the style police :-)

Reply:
@seamuslee001 fixed & passing